featuretools/tests/primitive_tests/test_feature_serialization.py

```diff
@@ -51,3 +55,24 @@ def test_pickle_features_with_custom_primitive(es):
 
     assert any([isinstance(feat.primitive, NewMax) for feat in features_original])
     pickle_features_test_helper(asizeof(es), features_original)
+
+
+def test_serialized_renamed_features(es):
```

Comment:
We should put all the types of features in `Identity feature, Transform feature, Aggregation feature, and Direct feature` in 1 feature list, and then rename each feature, and see if they were correctly serialized, and deserialized.  Let's not separate them into `test_agg_feats.py`, `test_direct_features`, `test_identify_features.py`, and `test_transform_features.py`